proto/qdrant/cloud/quota/v1/quota.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package qdrant.cloud.quota.v1;
+
+import "buf/validate/validate.proto";
+import "google/api/annotations.proto";
+import "qdrant/cloud/common/v1/common.proto";
+
+// Quota is the API used to configure Quotas.
+service QuotaService {
+  // Get quotas for the authenticated user.
+  // Required permissions:
+  // - None (authenticated only)
+  rpc GetAuthenticatedUserQuotas(GetAuthenticatedUserQuotasRequest) returns (GetAuthenticatedUserQuotasResponse) {
+    // permissions
+    option (common.v1.permissions) = "";
+    // custom account-id expression
+    option (qdrant.cloud.common.v1.account_id_expression) = "";
+    // custom supported-actor-types expression. (Requires to be authenticated as a user)
+    option (qdrant.cloud.common.v1.supported_actor_types) = ACTOR_TYPE_USER;
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/quota/v1/users/me/quotas"};
+  }
+  // Get quotas for the account identified by the given account ID.
+  // Required permissions:
+  // - read:account
+  rpc GetAccountQuotas(GetAccountQuotasRequest) returns (GetAccountQuotasResponse) {
+    // permissions
+    option (common.v1.permissions) = "read:account";
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/quota/v1/accounts/{account_id}/quotas"};
+  }
+}
+
+// buf:lint:ignore QDRANT_CLOUD_REQUIRED_REQUEST_FIELDS
+// GetAuthenticatedUserQuotasRequest is the request for the GetAuthenticatedUserQuotas function
+message GetAuthenticatedUserQuotasRequest {
+  // Empty
+}
+
+// GetAuthenticatedUserQuotasResponse is the response from the GetAuthenticatedUserQuotas function
+message GetAuthenticatedUserQuotasResponse {
+  // Maximum number of accounts this user can be the owner of
+  uint32 max_owned_accounts = 1;
```

Comment:
// A value of 0 means unlimited. ??

Reply:
Maybe you can add a note like:
// To get the actual list invoke AccountService.ListAccounts